binding/android/.idea/misc.xml

```diff
@@ -5,11 +5,12 @@
     <option name="myDefaultNotNull" value="android.support.annotation.NonNull" />
```

Comment:
I don't think we should push this file to repo. Are we ignoring the whole .idea directory in .gitignore?

Reply:
No. only these two sub-folders:

/.idea/workspace.xml
/.idea/libraries

why?